data/progress.yml

```diff
@@ -21855,7 +21855,7 @@ ports:
   import_statement: ''
   module_native: ✅
   modules_support_date: 2025/5/15
-  name: inifinity
```

Comment:
progress.yml is auto generated. If you want to change something, goto `progress_overwrite.yml` or `raw_progress.yml`